ui/redux/selectors.js

```diff
@@ -262,6 +262,25 @@ export const getParsedLocusList = createSelector(
       })
       locusList.items.sort(compareObjects('display'))
       locusList.rawItems = locusList.items.map(({ display }) => display).join(', ')
+
+      if (locusList.paLocusList) {
```

Comment:
this selector is used in a lof of other places where this computation would not be helpful, so I think this isn't a great place to add it. Maybe add it to the `onChange` of the `LocusListSelector` instead